xsd/netex_part_1/part1_ifopt/netex_ifopt_path_version.xsd

```diff
@@ -1,6 +1,7 @@
 <?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Gutting ifopt like this should have been done AFTER a group discussion. Not happy.